apps/issuer/models.py

```diff
@@ -1712,7 +1710,8 @@ def save(self, *args, **kwargs):
             if blacklist.api_query_is_in_blacklist(
                 self.recipient_type, self.recipient_identifier
             ):
-                logger.event(badgrlog.BlacklistAssertionNotCreatedEvent(self))
+                logger.warning("The recipient '%s' is in the blacklist for this ('%s') badge class",
+                               self.recipient_identifier, self.badgeclass.entity_id)
```

Comment:
In other files, information like the username or emails where logged with at most `info` level.
The `recipient_identifier` seems to be an email. Shouldn't we either strip some characters from it or not log it in warning? (Generally speaking, some sort of stripping mechanism is an interesting idea :thinking: )